doc/orchestrator.rst

```diff
@@ -165,3 +165,460 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+This function returns a ``RunSettings`` object.
+Here, we specify the path to the source code file (script) to
+``exe_args`` and the run command to ``exe``.
+
+.. code-block:: python
+
+  model_settings = exp.create_run_settings(exe="python", exe_args="./model_multidb_example.py")
+
+With the ``RunSettings`` instance,
+configure the the distribution of computational tasks (``RunSettings.set_nodes()``) and the number of instances
+the script is execute on each node (``RunSettings.set_tasks_per_node()``). In this
+example, we specify to SmartSim that we intend to execute the script once on a single node.
+
+.. code-block:: python
+
+  model_settings.set_nodes(1)
+  model_settings.set_tasks_per_node(1)
+
+Step 2: Initialize
+""""""""""""""""""
+Next, create a ``Model`` instance using the ``Experiment.create_model()``.
+Pass the ``model_settings`` object as an argument
+to the ``create_model()`` function and assign to the variable ``model``.
+
+.. code-block:: python
+
+  model = exp.create_model("colocated_model", model_settings)
+
+Step 2: Colocate
+""""""""""""""""
+To colocate the model, use the ``Model.colocate_db_tcp()`` function to
+Colocate an Orchestrator instance with this Model over TCP/IP.
+
+.. code-block:: python
+
+  model.colocate_db_tcp()
+
+This method will initialize settings which add an unsharded
+database to this Model instance. Only this Model will be able
+to communicate with this colocated database by using the loopback TCP interface.
+
+Step 3: Start
+"""""""""""""
+Next, launch the colocated model instance using the ``Experiment.start()`` function.
+
+.. code-block:: python
+
+  exp.start(model, block=True, summary=True)
+
+.. note::
+  We set `block=True`,
+  so that ``Experiment.start()`` waits until the last Model has finished
+  before returning: it will act like a job monitor, letting us know
+  if processes run, complete, or fail.
+
+Clobber the databases
+^^^^^^^^^^^^^^^^^^^^^
+Finally, use the ``Experiment.stop()`` function to stop the database instances. Print the
+workflow summary with ``Experiment.summary()``.
+
+.. code-block:: python
+
+  exp.stop(single_shard_db, multi_shard_db)
+  logger.info(exp.summary())
+
+When you run the experiment, the following output will appear.
+
+.. code-block:: bash
+
+  00:00:00 system.host.com SmartSim[#####]INFO
+  |    | Name                         | Entity-Type   | JobID       | RunID   | Time    | Status    | Returncode   |
+  |----|------------------------------|---------------|-------------|---------|---------|-----------|--------------|
+  | 1  | colo_model                   | Model         | 1538905.8   | 0       | 3.5465  | Completed | 0            |
+  | 2  | single_shard_db_identifier_0 | DBNode        | 1538905.5   | 0       | 73.1798 | Cancelled | 0            |
+  | 3  | multi-shard-db-identifier    | DBNode        | 1538905.6+2 | 0       | 49.8503 | Cancelled | 0            |
+
+Source Code
```

Comment:
I think we need a quick section on how to run the example.

Reply:
I agree!

Reply:
these instructions assume that you have smartsim and smartredis install in your python env